contrib/config_option_database/utils/MergeYm.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+#############################################################################
+# Copyright (c) 2019 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+from pathlib import Path
+from sys import argv
+
+
+def get_grammar_files():
+    root_dir = Path(__file__).resolve().parents[3]
+    files = list(root_dir.rglob('*.ym'))
```

Comment:
I'd answer it, since I just left here a comment: 
good question, but the generated grammar files have '.y' extension, e.g. `build/modules/afsocket/afsocket-grammar.y`.

However, it would be good to include only `modules` dir, or exclude the file `./dev-utils/plugin_skeleton_creator/plugin_template_grammar.ym`.

Reply:
Done, good idea, thanks! :)